tools/cluster_boxes.py

```diff
@@ -53,12 +60,50 @@ def main(args):
         for box in boxes:
             w = box[2] - box[0] + 1
             h = box[3] - box[1] + 1
-            if w > args.min_box_size[0] and h > args.min_box_size[1]:
+            if w > min_box_size[0] and h > min_box_size[1]:
                 wh_stats.append((w, h))
 
+    return wh_stats
+
+
+def get_sizes_from_coco(annotation_path, root, target_image_wh, min_box_size):
+    import imagesize
```

Comment:
Do we have it in requirements? If not please add it to optional ones.

Reply:
added